crates/prover/Cargo.toml

```diff
@@ -10,6 +10,7 @@ repository.workspace = true
 
 [dependencies]
 # Citrea Deps
+bitcoin-da = { path = "../bitcoin-da" }
```

Comment:
remove

Reply:
that's there because of compress blob func